lib/kubeclient/config.rb

```diff
@@ -131,11 +130,16 @@ def fetch_context(context_name)
       [cluster, user, namespace]
     end
 
-    def fetch_cluster_ca_data(cluster)
+    def cluster_ca_data?(cluster)
+      cluster.key?('certificate-authority') || cluster.key?('certificate-authority-data')
+    end
+
+    def populate_cert_store_from_cluster_ca_data(cluster, cert_store)
       if cluster.key?('certificate-authority')
-        File.read(ext_file_path(cluster['certificate-authority']))
+        cert_store.add_file(ext_file_path(cluster['certificate-authority']))
       elsif cluster.key?('certificate-authority-data')
-        Base64.decode64(cluster['certificate-authority-data'])
+        ca_cert_data = Base64.decode64(cluster['certificate-authority-data'])
+        cert_store.add_cert(OpenSSL::X509::Certificate.new(ca_cert_data))
```

Comment:
Umm, looks like the fix only covered `certificate-authority`, but `certificate-authority-data` would still have the bug.
- [ ] need to add test with concatenated inline data.
- [ ] can we fix it?
- [ ] if not, correct CHANGELOG to explain partial fix.